scrapy/utils/python.py

```diff
@@ -74,9 +74,9 @@ def unicode_to_str(text, encoding=None, errors='strict'):
 
     if encoding is None:
         encoding = 'utf-8'
-    if isinstance(text, unicode):
+    if isinstance(text, six.text_type):
         return text.encode(encoding, errors)
-    elif isinstance(text, str):
+    elif isinstance(text, bytes):
         return text
     else:
         raise TypeError('unicode_to_str must receive a unicode or str object, got %s' % type(text).__name__)
```

Comment:
I think "unicode or str" should be changed to "unicode, str or bytes" or "unicode or bytes"